spec/audited/rspec_matchers_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "spec_helper"
+
+describe Models::ActiveRecord::UserExceptPassword do
+  let(:non_audited_columns) { subject.class.non_audited_columns }
+
+  it { should_not be_audited.only(non_audited_columns) }
```

Comment:
I'm trying to drive home the point that these should work independently of each other. Might be helpful to have a case chaining them together as well.